index.html

```diff
@@ -0,0 +1,168 @@
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">Back End Dev</h4>
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">2015</h4>
+      </div>
+        <p class="highlights">
+          A daily selection of privately personalized reads no accounts for
+          sign-ups required.
+        </p>
+        <button class="tag1">HTML</button>
+        <button class="tag2">CSS</button>
+        <button class="tag3">Javascript</button>
+      
+        <button class="project-detail">See Project</button>
+    </div>
+  </section>
+
+    <!-- Third project -->
+  <section class="work-page-right">
+   <div class="card3-image">
+    <img src="/images/works/multi_post_story/ProfessionalArt_printing.svg" alt="Poster for Uber navigation" style="width: 100%;"/>
+   </div>
+    <div class="card3-content">
+      <h3 class="work-title">Multy-Post Stories</h3><br><br>
+      <div class="clients">
+        <h4>CANOPY</h4>
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">Back End Dev</h4>
+        <img class="img-clients" src="images/works/Counter.svg" alt="">
+        <h4 style="color: #6b778c">2015</h4>
+      </div>
+        <p class="highlights">
+          A daily selection of privately personalized reads no accounts for
+          sign-ups required.
+        </p>   
+          <button class="tag1">HTML</button>
+          <button class="tag2">CSS</button>
+          <button class="tag3">Javascript</button>
+
+        <button class="project-detail">See Project</button>
+    </div>
+  </section>
+
+<!-- About myself section -->
+<section id="about-me" class="about-myself">
+  <div class="about-myself-content">
+      <h2 class="about-myself-title">
+        About <br />
+        Myself
+      </h2>
+      <p class="myself-content">
+        Hello I am a software develper. I can help you build a product, feature
+        or website. Look through some of my work and expereince. If you like
+        what you see and have a project you need coder, don't husitate to
+        contact me.
+      </p>
+      <h3 class="links">LET'S CONNECT</h3>
+      <ul class="socials">
+        <li><a href="#" class="fa fa-facebook"></a></li>
+        <li><a href="#" class="fa fa-twitter"></a></li>
+        <li><a href="#" class="fa fa-linkedin"></a></li>
+        <li><a href="#" class="fa fa-google"></a></li>
+        <li><a href="#" class="fa fa-github"></a></li>
+        <li><a href="#" class="fa fa-javascript"></a></li>
+      </ul>
+      <button class="resume">Get My Resume</button>
+  </div>
+  <div class="about-myself-languages">
+      <h3 class="languages">Languages</h3>
+      <div class="framework-skills">
+        <img src="images/skills/html.svg" alt="HTML skill" />
+        <img src="images/skills/css.svg" alt="CSS skill" />
+        <img src="images/skills/js.svg" alt="Javascript skill" />
```

Comment:
Please use ul>li tags for the technologies here for semantic reasons. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`